src/main/java/org/jboss/tools/intellij/openshift/tree/application/ApplicationsTreeStructure.java

```diff
@@ -213,15 +216,19 @@ private Object[] getRegistryComponentTypes(DevfileRegistryNode element) {
         List<DevfileRegistryComponentTypeNode> result = new ArrayList<>();
         try {
             element.getRoot().getOdo().getComponentTypes(element.getName()).forEach(type -> result.add(new DevfileRegistryComponentTypeNode(root, element, type)));
-        } catch (IOException e) {}
+        } catch (IOException e) {
+        }
```

Comment:
Shouldn't we actu upon the exception? display that registry discovery failed?

Reply:
done.